test/api/v3/integration/user/auth/POST-register_local.test.js

```diff
@@ -288,6 +288,21 @@ describe('POST /user/auth/local/register', () => {
       });
     });
 
+    it('sanitizes email params to a lowercase string before creating the user', async () => {
+      let username = generateRandomUserName();
+      let email = 'ISANEmAiL@GMAIL.coM';
```

Comment:
best to use @example.net here rather than @gmail.com, just in case someone has that email address and doesn't like it being used in github